cinema/views.py

```diff
@@ -0,0 +1,43 @@
+from django.shortcuts import get_object_or_404
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from cinema.models import Movie
+from cinema.serializes import MovieSerializer
+
+
+@api_view(["GET", "POST"])
+def movie_list(request):
+    if request.method == "GET":
+        movies = Movie.objects.all()
+        serializer = MovieSerializer(movies, many=True)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+    else:
+        serializer = MovieSerializer(data=request.data)
+        if serializer.is_valid():
+            serializer.save()
```

Comment:
The `MovieSerializer` should ideally be a `ModelSerializer` to automatically handle the `save()` method. Ensure that the `MovieSerializer` is updated to extend `serializers.ModelSerializer` for better functionality.